beats/purge.sls

```diff
@@ -1,3 +1,4 @@
+{% if salt['pillar.get']('beats:use_upstream_repo', 'True') %}
 {% set version = salt['pillar.get']('beats:version', '5') %}
```

Comment:
The current branch is `7.x`, so default `beats:version` should be `7`. The same should be done in

https://github.com/saltstack-formulas/beats-formula/blob/53bd91688bc8cbba9ac2d43ddab4e11fcd3fd14a/beats/repository.sls#L6

Reply:
I was just thinking about the formula as a whole.  Version 5 of the elastic stack was updated as recently as March of 2019,  so it is still a valid default, also there are some breaking changes between versions 5 and 6 and I'm sure even more breaking changes from 6 to 7.  Seeing as the formula is known to work over all for 5, and I am not actively using this formula so I it would conflict with my current test environment I disagree with changing the default to version 7.

However, I still think that what I said about writing a defaults file would be a more effective way to change the defaults as they would not need to be changed everywhere in the formula and just in the defaults file.  It would take someone going through the whole formula and removing the current defaults and adding the reference to the merge of pillar and the defaults file.

Reply:
@absmith82 I think you are right, but not in this PR, can you create an issue for this so anybody who wants to join in can see what needs to be done.